internal/brokers/manager_test.go

```diff
@@ -239,8 +237,9 @@ func TestEndSession(t *testing.T) {
 		t.Run(name, func(t *testing.T) {
 			t.Parallel()
 
-			b, cfg := newBrokerForTests(t)
-			m, err := brokers.NewManager(context.Background(), nil, brokers.WithRootDir(filepath.Dir(cfg)), brokers.WithCfgDir(""))
+			cfgDir := t.TempDir()
```

Comment:
you are not reusing cggDirs here, but you do in the tests above (but attach that to the rootdir?). I think the 3 setup in those tests should be aligned.

Reply:
Oops, that was definitely a blunder. I'll fix it.

Reply:
Is is for withRootDir or rather brokers.WithCfgDir?

Reply:
It doesn't really matter, but I guess using `WithCfgDir` makes more sense